mne/io/brainvision/brainvision.py

```diff
@@ -358,10 +348,10 @@ def _get_vhdr_info(vhdr_fname, eog, misc, response_trig_shift, scale):
             highpass.append(line[5])
             lowpass.append(line[6])
         if len(highpass) == 0:
-            info['highpass'] = None
+            pass
         elif all(highpass):
             if highpass[0] == 'NaN':
-                info['highpass'] = None
+                pass
```

Comment:
Thanks for the comment, I was just about to complain :) It would be good to add a comment though because I could imagine in a few months going through here and errantly removing this clause.